src/Source/AbstractPropertySourceIterator.php

```diff
@@ -108,11 +84,11 @@ public function getDateTimeFormat(): string
     }
 
     /**
-     * @param object|mixed[] $current
+     * @phpstan-param object|array<mixed> $current
```

Comment:
imo it can remain as param

Reply:
Currently not https://github.com/rectorphp/rector/issues/7186